scripts/format_arpa_lm.py

```diff
@@ -44,6 +46,9 @@
 ngram_order = int(f.readline())
 f.close()
 
+# set the memory restriction for "sort"
+sort_mem_opt = ("--buffer-size={0} ".format(args.max_memory))
```

Comment:
I think we need to check whether args.max_memory is empty before set sort_mem_opt. 
Do you test the case if user doesn't specify the --max-memory option?